source/educators/how-tos/add_links_to_social_media_accounts.rst

```diff
@@ -17,8 +17,8 @@ To add links to your social media accounts to your profile, follow these steps.
    either your username or the URL to the profile page for each social media
    account that you want to add to your profile.
 
-   For example, if your Facebook username is ``edx``, you can enter either
-   ``edx`` or ``https://www.facebook.com/edx``.
+   For example, if your Facebook username is `` Oepn edX``, you can enter either
```

Comment:
There is a typo here, and the example isn't a great one (no one's username is 'Open edX' and 'Open edX' involves a space. Just change it to a generic example that doesn't use edx or Open EdX.

Reply:
done